Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -318,6 +318,11 @@ public extension String {
 		return replacingOccurrences(of: " ", with: "").replacingOccurrences(of: "\n", with: "")
 	}
 	
+    /// SwifterSwift: Check if the given string is blank
+    /// "test string".isBlank -> false
+    public var isBlank: Bool {
+        return self.trimmingCharacters(in: .whitespaces).isEmpty
```

Comment:
I think this would be better named as `isWhitespace`. Then include the character set of `.whiteSpaceAndNewLines`

Reply:
suggested changes incorporated in the latest commit. Please take a look.